dgbpy/dgbkeras.py

```diff
@@ -271,6 +336,12 @@ def epoch0endCB(epoch, logs):
   train_datagen = TrainingSequence( training, False, model, exfilenm=trainfile, batch_size=batchsize, scale=scale, transform=transform, tempnm=tempnm )
   validate_datagen = TrainingSequence( training, True, model, exfilenm=trainfile, batch_size=batchsize, scale=scale )
   nbchunks = len( infos[dgbkeys.trainseldicstr] )
+
+  if hasFastprogress() and not silent:
+    prog = ProgressBarCallback(train_datagen,validate_datagen)
+  else:
+    prog = ProgressNoBarCallback()
+
```

Comment:
add prog to the callbacks set here, once, before the loop over the chunks. Or does it have to be inside that for loop ?